targets/targets.json

```diff
@@ -1552,7 +1552,7 @@
             }
         },
         "detect_code": ["0765"],
-        "macros_add": ["USBHOST_OTHER"],
+        "macros_add": ["USBHOST_OTHER", "TWO_RAM_REGIONS"],
```

Comment:
I assume this is fine here but instead of macro, if this is a config that would result in MBED_CONF_MULTIPLE_RAM_REGIONS (if there are 3 , would it matter? if yes, then two is fine in the name) ?

Reply:
@0xc0170 the 'TWO' means that heap and stack are located in 2 separate ram regions.
It won't matter if you have 3 ram regions on the MCU. Only 2 will be concerned by this change.